starknet_py/hash/utils.py

```diff
@@ -23,9 +21,7 @@ def pedersen_hash(left: int, right: int) -> int:
     """
     One of two hash functions (along with _starknet_keccak) used throughout StarkNet.
     """
-    if use_cpp_variant():
-        return cpp_hash(left, right)
```

Comment:
`use_cpp_variant` function should be removed entirely, as I'm assuming it's no longer used anywhere.

Reply:
Left it, because I added deprecation to the pedersen hash.